Jenkinsfile

```diff
@@ -6,8 +6,16 @@ import com.tenable.jenkins.Constants
 import com.tenable.jenkins.builds.snyk.*
 import com.tenable.jenkins.builds.nexusiq.*
 
+def addParameters(final script, def theParams) {
+    if (script.env.BRANCH_NAME == 'master') {
+       theParams << script.booleanParam(defaultValue: false, description: 'Puplish to PYPI!', name: 'PUPBLISH2PYPI')
+    }
+    theParams
+}
+if (env.BRANCH_NAME == 'MASTER') {
```

Comment:
Please remove this. The build parameters are there to allow QG overwrites. They are valid on feature branches as well.